content/coronavirus_landing_page.yml

```
@@ -62,19 +62,18 @@ content:
   timeline:
     heading: Recent and upcoming changes
     list:
+    - heading: 27 October
+      paragraph: | 
+          Warrington moves to [Local COVID Alert Level: Very High](/guidance/local-covid-alert-level-very-high?priority-taxon=774cee22-d896-44c1-a611-e3109cce8eae)
       - heading: 25 October
```

A couple of spaces are needed here:

```suggestion
    list:
      - heading: 27 October
        paragraph: | 
            Warrington moves to [Local COVID Alert Level: Very High](/guidance/local-covid-alert-level-very-high?priority-taxon=774cee22-d896-44c1-a611-e3109cce8eae)
      - heading: 25 October
```